app/bundles/CampaignBundle/Executioner/ScheduledExecutioner.php

```diff
@@ -88,8 +88,9 @@ public function execute(Campaign $campaign, ContactLimiter $limiter, OutputInter
      * @throws Scheduler\Exception\NotSchedulableException
      * @throws \Doctrine\ORM\Query\QueryException
      */
-    public function executeByIds(array $logIds, OutputInterface $output = null)
+    public function executeByIds(array $logIds, OutputInterface $output = null, \DateTime $now = null)
     {
+        $now           = $now ?: new \DateTime();
```

Comment:
Just curious why isn't the class param considered anymore as it was on line 127?
```suggestion
        $now           = $now ?? $this->now ?? new \DateTime();
```